Junit/src/IPL.java

```diff
@@ -0,0 +1,364 @@
+import java.util.*;
+import java.io.*;
+import java.util.stream.*;
+
+
+class Player implements Comparable<Player>{
+
+    String name;
+    String team;
+    String role;
```

Comment:
Best practice would be to store the roles as an enum and use the enum constants wherever required instead of plain strings.